pom.xml

```
@@ -69,6 +69,7 @@
         <no-test-jar>false</no-test-jar>
         <useBeta>true</useBeta>
         <hpi.compatibleSinceVersion>2.26</hpi.compatibleSinceVersion>
+      <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
```

nit: Looks like the indentation is different than the previous line:

```suggestion
        <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
```